addons/hr_expense/models/res_company.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    expense_extract_show_ocr_option_selection = fields.Selection([
+        ('no_send', 'Do not digitalize bills'),
+        ('manual_send', "Digitalize bills on demand only"),
+        ('auto_send', 'Digitalize all bills automatically')], string="Send mode on expense attachments",
+        required=True, default='auto_send')
```

Comment:
Why is this field defined in here?
It is redefined and only used in `hr_expense_extract` (enterprise) module.

@odony is it tolerated to remove it from saas~13.2?

Reply:
I had to go through it, here is some thoughts ...

I think it is linked to having the settings available directly in hr_expense and avoid waiting for installing hr_expense_extract + reload to change option.

For redefinition, I found out there were some differences in labelling.

Reply:
Don't remember exactly why I put it there. That's just needed to change the OCR modes so... not really needed here. Maybe I had got an error and putting this field here just stopped it.  If it work fine without ... we can delete it. And the hr_expense_extract module is installed by default with expense.

Reply:
> @odony is it tolerated to remove it from saas~13.2?

I guess, when it's caught during upgrade script review we can make an exception. Counts as a sacrificed kitten though.

Reply:
@KangOl  there it's what is needed to change to remove this thing from hr_expense https://github.com/odoo/odoo/pull/48176 and https://github.com/odoo/enterprise/pull/9416

Reply:
@KangOl @tde-banana-odoo @odony @tivisse @fda-odoo  so what are we doing with this ? we can change it by applying https://github.com/odoo/odoo/pull/48176 and https://github.com/odoo/enterprise/pull/9416 if you're ok with that. or do something else if you got another idea.

Reply:
For me everything is ok, I just want to avoid having required fields on company model when not necessary because of MC tests.

If you want to go with the attached PR, don't forget migration script.

Reply:
Ah sorry those are not stored fields actually. I would not do it in stable though.